# Change the value of the Version value in turpial.desktop to 1.0
# as 2.0 is not valid according to gentoo stuff.

diff --git a/turpial.desktop b/turpial.desktop
index e3d91f6..826122d 100755
--- a/turpial.desktop
+++ b/turpial.desktop
@@ -1,5 +1,5 @@
 [Desktop Entry]
-Version=2.0
+Version=1.0
 Name=Turpial
 Name[es]=Turpial
 GenericName=Microblogging client
